src/app/home-page/home-page.component.scss

```diff
@@ -0,0 +1,32 @@
+home-page-component {
+    .contentBackground {
```

Comment:
Class names shouldn't be camelCase.
https://stackoverflow.com/questions/1547986/can-i-use-camel-case-in-css-class-names